backend/src/test/java/ca/mcgill/purposeful/util/DatabaseUtil.java

```diff
@@ -32,6 +36,8 @@ public class DatabaseUtil {
 
   /** Method to clear the database completely @Author Shidan Javaheri */
   public void clearDatabase() {
+    collaborationRequestRepository.deleteAll();
```

Comment:
Awesome!

Reply:
request needs to be deleted first since if it is referencing a confirmation, it will be referencing something that doesn't exist, which creates foreign key issues

Reply:
Other PR merged if you want to do that :) thanks